.travis.yml

```diff
@@ -3,6 +3,12 @@ language: generic
 sudo: required
 dist: trusty
 install:
-    - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
+  - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
+  - npm install -g danger@alpha
+
 script:
-    - swift test
+  - swift test
+  # - swiftenv install 4.0
```

Comment:
Does Danger-Swift require Swift 4? Don't think `swiftenv` supports it yet (at least not when I tried a few weeks ago)